nixos/modules/services/networking/frp.nix

```diff
@@ -64,10 +64,8 @@ in
             Restart = "on-failure";
             RestartSec = 15;
             ExecStart = "${cfg.package}/bin/${executableFile} --strict_config -c ${configFile}";
-            StateDirectoryMode = lib.optionalString isServer "0700";
             DynamicUser = true;
             # Hardening
-            UMask = lib.optionalString isServer "0007";
```

Comment:
Switching these from `optionalString` to `mkIf` should have the same effect (depending on what type is merging these definitions) and is a more idiomatic use of the module system than `optionalAttrs` (when appropriate)